### PR TITLE
tasks: Add legacy URLs task.

### PR DESCRIPTION
**Pull request by timabbott**

### CODE REVIEW

**Comment by smarx** [comment]
*Automated message from Dropbox CLA bot*

@timabbott, it looks like you've already signed the Dropbox CLA. Thanks!

File: tasks/legacy_urls.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+the knowledge you learn with that task.
+
+## Background
+
+Zulip has a number of "legacy" API URLs that are not properly
+available via the Zulip REST API, which are present in
+`zproject/legacy_urls.py` in the main Zulip project.
+
+Please carefully read [https://github.com/zulip/zulip/issues/611](the
+master GitHub issue for this migration) for details on the migration
```
Comment (trueskawka) [inline tasks/legacy_urls.md]: The link and the description are reversed here, should be:
[the master GitHub issue for this migration](https://github.com/zulip/zulip/issues/611)

**Comment by showell** [comment]
One way I think we could enhance this is to ask the students to produce a screenshot of the new URL being called in run-dev.py, or some other way to validate that they manually tested it.  We can probably add that later, though.

This looks good to me.

**Comment by trueskawka** [comment]
Ensuring manual testing is a great idea!

**Comment by timabbott** [comment]
Merged, thanks folks!

**Pull request closed without merging**